src/olympia/lib/es/tests/test_commands.py

```diff
@@ -24,6 +24,9 @@ def setUp(self):
         # created during this test run.
         self.indices = self.es.indices.stats()['indices'].keys()
 
+        self.addons = []
+        self.expected = self.addons[:]
```

Comment:
I like the consistency, but this is just ` self.expected = []` 😄

Reply:
Yeah, I went back and forth changing `self.addons` in `setUp` instead of the test method and in the end left `self.expected` as is, eh, it's working :)